docs/docs/Agents/agent-tool-calling-agent-component.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Create a problem solving agent
+sidebar_position: 2
+slug: /agents-tool-calling-agent-component
+---
+
+Developing **agents** in Langchain is complex.
+
+The `AgentComponent` is a tool for easily creating an AI agent capable of reasoning through tasks using tools you provide.
+
+The component contains all of the elements you'll need for creating an Agent. Instead of managing LLM models and providers, pick your model and enter your API key. Instead of connecting a **Prompt** component, enter instructions in the component's **Agent Instruction** fields.
```

Comment:
```suggestion
The component contains all of the elements you'll need for creating an agent. Instead of managing LLM models and providers, pick your model and enter your API key. Instead of connecting a **Prompt** component, enter instructions in the component's **Agent Instruction** fields.
```